tasks/main.yml

```diff
@@ -14,6 +14,18 @@
 - import_tasks: setup_replace_user.yml
   when: raspi_config_replace_user['name'] != None
 
+- name: Verify if replace_user exists
+  stat:
+    path: "/etc/sudoers.d/{{ raspi_config_replace_user['name'] }}"
```

Comment:
Just a small remark - shouldn't this also include the pseudo-priority prefix, in line with how the directory is organized by default?